LDM-148.tex

```diff
@@ -941,42 +954,30 @@ \subsubsection{Alert Production Payload}\label{alert-production-payload}
 Master Calibration Images, Template Images, Prompt Products Database, and
 Calibration Database information from the Data Backbone.
 
-\subsubsection{MOPS Payload}\label{mops-payload}
+\subsubsection{Calibration Software}\label{calibration-software-payload}
```

Comment:
Seems like this should be a payload, or collection of payloads, for consistency with the other software described in this subsection.